frontend/src/pages/Explore/Explore.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { LoginContext } from '../../Components/LoginManager/LoginManager';
+import { getSightings } from '../../api/backendClient';
+import "./Explore.scss"
+
+interface SightingType {
+  username: string;
+  speciesName: string;
+  latitude: number;
+  longitude: number;
+  photoUrl: string;
+  description: string;
```

Comment:
shouldn't the `description` be nullable? it is in `SightingResponse.cs`

Reply:
? added for description now